csw-framework/src/main/scala/csw/framework/internal/wiring/Standalone.scala

```diff
@@ -26,7 +26,7 @@ object Standalone {
     import actorRuntime._
 
     val componentInfo = ConfigParser.parseStandalone(config)
-    val richSystem    = new CswFrameworkSystem(typedSystem)
+    val richSystem    = new CswFrameworkSystem(actorRuntime.actorSystem)
```

Comment:
actorSystem is already imported, can be used directly

Reply:
It is getting imported from two places, Earlier it was working because name as typedSystem at 1 place and actorSystem at another. Now I have made actorSystem everywhere, hence it was giving compilation error of ambiguity. 
import wiring._
import actorRuntime._